Account & Project/1. User guides/11. 檢視計畫資訊.md

```diff
@@ -1 +1,67 @@
+1. Did not apply for an account, please [<ins>sign up for an account</ins>](https://iservice.nchc.org.tw/nchc_service/nchc_member_apply_1.php)。
+2. Did not join any project that has enabled TWCC service, please [<ins>apply project</ins>](https://man.twcc.ai/@twccdocs/apply-project-and-credit-zh)==**Need to update:exclamation:**== , or notify the tenant admin to add you to the project.
+3. TWCC portal update, please clear the browser cache to sign in normally.
+:::
+
+### Step 2. View project information
+
+After sign in, you can view the project information on the overview page. The information includes:
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_a483ca66912a976c4758b729b7a43cd7.png)
+
+<span>&#10102;</span> User Identity: tenant admin or tenant user, in addition to the title, the screen picture are also different
+:::info
+<i class="fa fa-paperclip fa-20" aria-hidden="true"></i> **Note:** There are difference betwen tenant admin and users to use TWCC functions, please refer to [<ins>User permissions comparison</ins>](https://man.twcc.ai/@twccdocs/role-main-zh). ==**Need to update:exclamation:**==
+:::
+<span>&#10103;</span> Wallet and credit information, duration, system code
+    
+
+- **Tenant Admin** In addition to the personal credits, you may also view the remaining credit of the project and the total credit of the project.
+- Definition description:
+    - Duration: The effective period of the project.
+    - System code: The code planned in the system. If you encounter a problem, you can provide this information to the customer service staff.
+    - Wallet: When you use the service, the type of wallet for which the credit is deducted: main wallet or sub-wallet.
+        - Main wallet: the total credit of the project deducts the creidt of the sub wallet
+        - Sub-wallet: the usage credit allocated to members from the main wallet
+    - Credits Remaining: The remaining available credit.
+    - Total Credits: the accumulated credit obtained.
+
+<span>&#10104;</span> Service overview. Click "View Details>" to view three types of storage and resource usage:
+
+- Storage usage: Capacity of hyper file system (HFS) /home and /work directory
+    - Used capacity
+    - Available capacity
+    - Total Storage(Free or Purchased)
+- Compute, Analytics & Database: the number of resource services that have been established
+- Network & security: the number of resource services that have been established
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_f21ac33b7c4c15afc871d00dfc34b910.png)
+
+
+:::info
+<i class="fa fa-paperclip fa-20" aria-hidden="true"></i> **Note:** 
+For more project information (project members, wallet information, resource usage), please click the "Account & Project" link to [<ins>Member Center<i class="fa fa-question-circle fa-question-circle-for-service" aria-hidden="true"></i></ins>](https://man.twcc.ai/@twsdocs/howto-service-access-service-zh) to view.
```

Comment:
[SUG]

Please click the "Account & Project" and link to [<ins>Member Center<i class="fa fa-question-circle fa-question-circle-for-service" aria-hidden="true"></i></ins>](https://man.twcc.ai/@twsdocs/howto-service-access-service-zh) to view more project information (project members, wallet information, resource usage).